solr/core/src/java/org/apache/solr/core/NodeConfig.java

```diff
@@ -219,8 +218,7 @@ public static NodeConfig loadNodeConfig(Path solrHome, Properties nodeProperties
         if (zkClient.exists("/solr.xml", true)) {
           log.info("solr.xml found in ZooKeeper. Loading...");
           byte[] data = zkClient.getData("/solr.xml", null, null, true);
-          return SolrXmlConfig.fromInputStream(
```

Comment:
This change does not use "data" that was just fetched; surely it can't be correct?